README.md

```diff
@@ -42,6 +42,23 @@ module "vpc" {
 
         subnet-02 = []
     }
+    
+    routes = [
+        {
+            name             = "egress-inet"
+            description      = "route through IGW to access internet"
+            dest_range       = "0.0.0.0/0"
+            tags             = "egress-inet"
+            next_hop_gateway = "true"
+        },
+        {
+            name        = "app-proxy"
+            description = "route through proxy to reach app"
+            dest_range  = "10.50.10.0/24"
+            tags        = "app-proxy"
+            next_hop_ip = "10.10.20.10"
```

Comment:
Is using IPs considered a best practice for routing within GCP or should customers generally use instance names? Let's be prescriptive here.